### PR TITLE
Line to track the finale

### PR DESCRIPTION
**Pull request by rpauszek**
**Why this PR?**

This is the final PR to align the "line" vs "track" terminology, focused on the kymotracker widget

Docs build [here](https://lumicks-pylake.readthedocs.io/en/line_to_track_the_finale/examples/cas9_kymotracking/cas9_kymotracking.html)

### CODE REVIEW

File: lumicks/pylake/nb_widgets/kymotracker_widgets.py [hunk 1]
```diff
@@ -80,12 +81,48 @@ def __init__(
...
+            Collection of detected tracks.
+        """
+        return self._tracks_history.state
 
     @lines.setter
+    @deprecated(
+        reason=(
+            "This property will be removed in a future release. Use the `tracks` property instead."
+        ),
+        action="always",
```
Comment (JoepVanlier) [inline lumicks/pylake/nb_widgets/kymotracker_widgets.py:117]: 👍

File: changelog.md [hunk 2]
```diff
@@ -44,6 +44,7 @@
 * The property `sample_rate` of `Continuous` data now returns a `float` instead of an `int``
 * Removed deprecated argument `roi` from `CorrelatedStack.export_tiff`. Use `CorrelatedStack.crop_by_pixels()` to select the ROI before exporting.
 * `KymoWidgetGreedy` now enforces using keywords for all arguments after the first two (`kymo` and `channel`).
+* The `KymoWidgetGreedy` attributes/function `plotted_lines`, `show_lines`, `adding` and `update_lines()` have been removed (replaced with private API).
```
Comment (JoepVanlier) [inline changelog.md:47]: 👍

File: lumicks/pylake/nb_widgets/kymotracker_widgets.py [hunk 3]
```diff
@@ -277,29 +277,53 @@ def _update_tracks(self):
...
         except (RuntimeError, IOError) as exception:
             self._set_label("status", str(exception))
 
+    @deprecated(
+        reason=(
+            "This property will be removed in a future release. Use the `tracks` property instead."
+        ),
+        action="always",
+        version="0.13.0",
+    )
```
Comment (JoepVanlier) [inline lumicks/pylake/nb_widgets/kymotracker_widgets.py:298]: Parameters will through a warning on the docs build :(

File: lumicks/pylake/nb_widgets/kymotracker_widgets.py [hunk 4]
```diff
@@ -277,29 +277,53 @@ def _update_tracks(self):
...
         delimiter : str
             Which delimiter to use in the csv file.
         sampling_width : int or None
-            When supplied, this will sample the source image around the kymograph line and export
+            When supplied, this will sample the source image around the kymograph track and export
             the summed intensity with the image. The value indicates the number of pixels in either direction
             to sum over.
         """
         self.tracks.save(filename, delimiter, sampling_width)
+        self.tracks.save(filename, delimiter, sampling_width)
```
Comment (JoepVanlier) [inline lumicks/pylake/nb_widgets/kymotracker_widgets.py:330]: Rebase mishap?

File: docs/examples/cas9_kymotracking/cas9_kymotracking.rst [hunk 5]
```diff
@@ -130,7 +130,17 @@ in the track you wish to connect it to.
...
+    kymowidget = lk.KymoWidgetGreedy(
+        kymo_ds,
+        "green",
+        axis_aspect_ratio=0.5,
+        min_length=4,
+        pixel_threshold=3,
+        window=6,
+        sigma=0.14,
+        vmax=8,
+        cmap="viridis"
```
Comment (JoepVanlier) [inline docs/examples/cas9_kymotracking/cas9_kymotracking.rst:142]: :)

**Review by JoepVanlier — CHANGES_REQUESTED** [review]
Couldn't find anything controversial in this one. Only request is to fix a small duplicate line :)

File: lumicks/pylake/nb_widgets/kymotracker_widgets.py (hunk 4 above)
Comment (rpauszek) [reply lumicks/pylake/nb_widgets/kymotracker_widgets.py:330]: 😒

**Review by JoepVanlier — APPROVED** [review]
Looks good to me.

**Pull request merged**